tools/jdbc/src/main/java/org/duckdb/DuckDBResultSetMetaData.java

```diff
@@ -150,8 +154,21 @@ public String getColumnClassName(int column) throws SQLException {
 			return Date.class.toString();
 		case Types.TIMESTAMP:
 			return Timestamp.class.toString();
+		case Types.TIME_WITH_TIMEZONE:
+			return OffsetDateTime.class.toString();
 		case Types.BLOB:
 			return DuckDBBlobResult.class.toString();
+		case Types.DECIMAL:
+			switch (column_types_meta[column].type_size) {
```

Comment:
Should decimals return a `short` or `integer` class? Internally this is what they use but within Java not no?

Reply:
Yes, you are right. Thanks for catching this. It should always return BigDecimal.